src/it/resources/org/checkstyle/suppressionxpathfilter/methodcount/SuppressionXpathRegressionMethodCount3.java

```diff
@@ -0,0 +1,6 @@
+package org.checkstyle.suppressionxpathfilter.methodcount;
+
+class SuppressionXpathRegressionMethodCount3 { // warn
```

Comment:
this looks like the same file as `SuppressionXpathRegressionMethodCount1` - could that one be reused instead?

Reply:
Yes! You're right. I'll just fix it.

Reply:
Done! @esilkensen

Reply:
Cool, thanks. Code changes and diff report look good - now that the 3rd test input file is removed, the numbers on the 4th 5th could be decremented.

Alternatively, what do you think about a single test input file with public/private/protected/default methods, and separate test cases for each type of method - would that work?

Reply:
Yes, that will work. But a look at https://github.com/checkstyle/checkstyle/issues/7987. I think it expects two or more. What do you say?

Reply:
Or is it only for test methods and not input files?

Reply:
I think Roman’s comment there refers to having two or more test methods, but if you want to rename the 4th and 5th input files to 3 and 4 respectively, I am good to approve PR. Maybe another reviewer has an opinion about combining them into one file - either way seems ok to me.

Reply:
Done! Thanks for the review.